sklearn/calibration.py

```diff
@@ -1267,8 +1267,9 @@ def from_estimator(
         if not is_classifier(estimator):
             raise ValueError("'estimator' should be a fitted classifier.")
 
```

Comment:
This check can be removed now since ``_get_response_values_binary`` will handle it